playwright/tests/organize/campaign-detail/action-buttons.spec.ts

```diff
@@ -69,17 +67,37 @@ test.describe('Campaign action buttons', async () => {
                 },
             });
 
-            // Edit task
+            // Edit title
             await page.fill('#title', newTitle);
+
+            // Set manager
+            await page.click('[name=manager_id]');
+            await page.fill('[name=manager_id]', 'Rosa');
+            await page.click('text="Rosa Luxemburg"');
+
+            // Submit the form
             await page.click('button > :text("Submit")');
 
             // Check that title changes on page
-            const taskTitle = page.locator('data-testid=page-title');
-            await expect(taskTitle).toContainText(newTitle);
+            const campaignTitle = page.locator('data-testid=page-title');
+            await expect(campaignTitle).toContainText(newTitle);
+
+            // Check that patch was made correctly
+            const log = await moxy.logRequests();
+            const patchRequest = await log.log.find(req =>
+                req.method === 'PATCH' &&
+                req.path === '/v1/orgs/1/campaigns/1',
+            );
```

Comment:
Thanks for adding these, was recently thinking about how we weren't testing requests when we built this.